### ISSUE TITLE
Option to prefer adding portable packages to PATH even if user have admin rights / developer mode enabled

### ISSUE THREAD

**Issue by soredake** [microsoft/winget-cli#4053]
### Description of the new feature / enhancement

One reason why i want this is because winget creates all links with .exe extension even if you specify `.ps1` for example.

Example package with `.ps1` alias https://github.com/soredake/winget-pkgs/blob/LesFerch.WinSetView-2.80-02770D614FEFB4283B8C4F006A105224A0E3E019/manifests/l/LesFerch/WinSetView/2.80/LesFerch.WinSetView.installer.yaml#L11-L12

Also this https://github.com/microsoft/winget-cli/issues/2711

**Comment by Trenly** [comment]
I don’t think that .ps1 will ever be supported, as it goes against the policy of not using scripts as installers; But perhaps this is different because its a portable? @denelon would probably know.

There are issues open to support other portable types such as .jar files, but they likey wouldn’t be impacted by symlinking.

I do agree that being able to specify to always perform path cramming could be a useful setting, but if many portable packages are installed it could lead to the environment variable going over the character limit

**Comment by soredake** [comment]
> I don’t think that .ps1 will ever be supported, as it goes against the policy of not using scripts as installers

It's not an installer, it's just an alias:
![image](https://github.com/microsoft/winget-cli/assets/5204968/e318978e-6029-4b3d-99ac-a0f460314f38)

**Comment by Trenly** [comment]
> It's not an installer, it's just an alias:

But the file path still points to a .ps1, which is what I was referring to.
